Can't resolve '@scure/bip39/wordlists/english' ... because it was resolved as fully specified

This is the error I'm getting right after installing the module and trying to generate private key:

ERROR in ./node_modules/nostr-tools/lib/esm/nostr.mjs 1206:0-58
Module not found: Error: Can't resolve '@scure/bip39/wordlists/english' in '/Users/alex/projects/nostr-client/node_modules/nostr-tools/lib/esm'
Did you mean 'english.js'?
BREAKING CHANGE: The request '@scure/bip39/wordlists/english' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

--- a/package.json
+++ b/package.json
@@ -19,10 +19,10 @@
   },
   "license": "Unlicense",
   "dependencies": {
-    "@noble/curves": "1.0.0",
-    "@noble/hashes": "1.3.0",
+    "@noble/curves": "1.1.0",
+    "@noble/hashes": "1.3.1",
     "@scure/base": "1.1.1",
-    "@scure/bip32": "1.3.0",
+    "@scure/bip32": "1.3.1",
     "@scure/bip39": "1.2.1"
   },
   "keywords": [

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "@noble/hashes": "1.3.0",
     "@scure/base": "1.1.1",
     "@scure/bip32": "1.3.0",
-    "@scure/bip39": "1.2.0"
+    "@scure/bip39": "1.2.1"
   },
   "keywords": [
     "decentralization",

